InnerEye/ML/model_testing.py

```diff
@@ -437,8 +440,27 @@ def test_epoch(test_epoch: int, run_recovery: Optional[RunRecovery]) -> Optional
         else:
             results[epoch] = epoch_result
 
+            if isinstance(epoch_result, ScalarMetricsDict):
+                epoch_folder = config.outputs_folder / get_epoch_results_path(epoch, data_split, model_proc)
+                csv_file = epoch_folder / METRICS_FILE_NAME
+
+                logging.info(f"Writing {data_split.value} metrics to file {str(csv_file)}")
+
+                # if we are running on the validation set, these may have been written during train time
```

Comment:
Add something like "else if running on the test set, create the metrics csv" ?

Reply:
Changed the comment to clarify.